packages/modular-scripts/src/cli.ts

```diff
@@ -343,6 +339,44 @@ async function build(directoryName: string, preserveModules?: boolean) {
   }
 }
 
+interface NPMRegistryListing {
+  "dist-tags": Record<string, string>;
+}
+
+async function preflight() {
+  if (process.env.SKIP_PREFLIGHT_CHECK !== 'true') {
+    const { stdout: registry } = await execa('yarnpkg', ['config', 'get', 'registry']);
+
+    const url = String(new URL("/modular-scripts", registry));
+
+    const res = await fetch(url);
+    const modualarScriptsRegistry: NPMRegistryListing = await res.json();
+    const newVersion = modualarScriptsRegistry["dist-tags"]["latest"];
+
+    const { version }: ModularPackageJson = fs.readJSONSync(path.join(__dirname, "..", "package.json"));
+    if (newVersion != version) {
+      const { dependencies }: ModularPackageJson = fs.readJSONSync(path.join(getModularRoot(), "package.json"));
+
+      const update  = await prompts({
+        type: 'confirm',
+        name: 'value',
+        message: `Modular is out of date - would you like to update from ${version} to ${newVersion}`,
```

Comment:
`modular` is lowercase. Also a suggestion for the wording. 
```suggestion
        message: `Your version of modular is ${version} . Would you like to update to ${newVersion}?`,
```